oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2RegisterDAO.java

```diff
@@ -47,6 +49,18 @@ public H2RegisterDAO(JDBCHikariCPClient h2Client, StorageBuilder<RegisterSource>
         return (RegisterSource)getByID(h2Client, modelName, id, storageBuilder);
     }
 
+    @Override
+    public Map<String, RegisterSource> batchGet(String modelName, String... ids) throws IOException {
+        Map<String, RegisterSource> map = new HashMap<>(ids.length);
+        for (String id : ids) {
+            RegisterSource registerSource = this.get(modelName, id);
```

Comment:
We can do real batch queries as ES do here.

Reply:
@peng-yongsheng what do you think?

Reply:
Now, this has been modified to be a real batch query.